app/navigation/root-navigator.tsx

```diff
@@ -441,10 +450,34 @@ export const MoveMoneyNavigator: NavigatorType = () => (
       name="moveMoney"
       component={MoveMoneyScreenDataInjected}
       // options={{ title: translate("MoveMoneyScreen.title") }}
-      options={{
-        headerShown: false,
-        title: translate("MoveMoneyScreen.title"),
-      }}
+      options={({ navigation }) => {
+        return {
+          headerShown: true,
+          title: "", //translate("MoveMoneyScreen.title"),
```

Comment:
why is the title replaced with an empty string? (just curious, seems like it was just non-existent before)

we also have the title on line 452 commented out inside of its own options hash and then again here commented out.  i don't believe we should leave commented code in production unless it's for documentation purposes. can we remove these?